README.md

```diff
@@ -16,40 +17,43 @@ The API of this project is still evolving. The use of vendoring tool is recommen
 
 # Quickstart
 
-By completing this quickstart, you will learn how to trace local methods. For more details, please view 
+By completing this quickstart, you will learn how to trace local methods. For more details, please view
 [the example](example_trace_test.go).
 
 ## Configuration
 
-GO2Sky can export traces to Apache SkyWalking OAP server or local logger. In the following example, we configure GO2Sky to export to OAP server, 
-which is listening on `oap-skywalking` port `11800`, and all the spans from this program will be associated with a service name `example`. 
-`reporter.GRPCReporter` can also adjust the behavior through `reporter.GRPCReporterOption`, [view all](docs/GRPC-Reporter-Option.md).
- 
+GO2Sky can export traces to Apache SkyWalking OAP server or local logger. In the following example, we configure GO2Sky
+to export to OAP server, which is listening on `oap-skywalking` port `11800`, and all the spans from this program will
+be associated with a service name `example`.
+`reporter.GRPCReporter` can also adjust the behavior through `reporter.GRPCReporterOption`
+, [view all](docs/GRPC-Reporter-Option.md).
+
 ```go
 r, err := reporter.NewGRPCReporter("oap-skywalking:11800")
 if err != nil {
-    log.Fatalf("new reporter error %v \n", err)
+log.Fatalf("new reporter error %v \n", err)
 }
 defer r.Close()
 tracer, err := go2sky.NewTracer("example", go2sky.WithReporter(r))
-// create with sampler
-// tracer, err := go2sky.NewTracer("example", go2sky.WithReporter(r), go2sky.WithSampler(0.5))
```

Comment:
Why remove this?

Reply:
It has been described here

https://github.com/SkyAPM/go2sky/blob/0f9874e2d30cb5e4b3e4721c0830c52eb8c0f97f/README.md?plain=1#L39-L43